project/AppDependencies.scala

```diff
@@ -2,20 +2,20 @@ import sbt.*
 
 object AppDependencies {
 
-  val hmrcMongoVersion = "1.4.0"
-  val bootstrapVersion = "7.23.0"
+  val hmrcMongoVersion = "1.5.0"
+  val bootstrapVersion = "8.0.0"
```

Comment:
the platform is not ready and not recommending to move to bootstreap v 8 you can see details here https://hmrcdigital.slack.com/archives/C0GS60DK2/p1700738356825869?thread_ts=1700733425.496229&cid=C0GS60DK2